src/shared/constants.h

```diff
@@ -63,6 +63,9 @@ const double DRIBBLER_RPM_TO_RADIO_CONVERSION_FACTOR = 1.0 / 300.0;
 
 const double POSSESSION_TIMESTAMP_TOLERANCE_IN_MILLISECONDS = 10;
 
+// The maximum number of elements in the circular buffer for the motion controller
+const size_t CIRCULAR_BUFFER_MAX_NUM_ELEMENTS = 10;
```

Comment:
I don't think we need this constant. It's very specific to the motion controller but the buffer is a generic structure that anything could use. The controller should define its own internal constant if it needs it.

Reply:
The reason for this constant was because the initial ticket seemingly referred to a constant.
"Circular buffer must be able to contain up to CIRCULAR_BUFFER_MAX_NUM_ELEMENTS (10) elements"

I'll remove it for now assume if need be, whoever is using the circular buffer can add in the constant when they need it.

Reply:
Didn't realize it was part of the ticket, but I think just making the size a parameter is the way to go here since it's a generic datastructure.